### ISSUE TITLE
Module build failed: Duplicate declaration "client"

### ISSUE THREAD

**Issue by pSchaub** [nuxt-community/apollo-module#53]
nuxt.config.js
```
  apollo: {
    clientConfigs: {
      default: '~/apollo/client-configs/client1.js',
      client2: '~/apollo/client-configs/client2.js'
    }
  },
```

Error:
```
 ERROR  Failed to compile with 1 errors                                 10:35:11

 error  in ./.nuxt/apollo.plugin.0bafa84e.js

Module build failed: Duplicate declaration "client"

  41 | 
  42 |   
> 43 |     let client = require('~/apollo/client-configs/client2.js')
     |         ^
  44 |     // es6 module default export or not
  45 |     client = client.default(ctx) || client(ctx)
  46 |     const cache = client.cache || new InMemoryCache()


 @ ./.nuxt/index.js 286:0-98
 @ ./.nuxt/client.js
 @ multi webpack-hot-middleware/client?name=client&reload=true&timeout=30000&path=/__webpack_hmr ./.nuxt/client.js
```

![bildschirmfoto von 2017-12-22 10-36-41](https://user-images.githubusercontent.com/11846736/34293173-06e36202-e704-11e7-9ad6-14042e241084.png)

https://github.com/nuxt-community/apollo-module/blob/master/plugin.js#L18

**Comment by dohomi** [comment]
Might  be connected to this PR: https://github.com/nuxt-community/apollo-module/pull/37

**Comment by dohomi** [comment]
@pSchaub can you validate if #66 would fix your current problem? We are verifying that PR at the moment

**Comment by dohomi** [comment]
@pSchaub could you review if #66 is solving your issue? Thanks

**Comment by yaki4** [comment]
@Dohomi This solution could come really handy and could fix this problem (at least for me)

**Comment by dohomi** [comment]
This should be fixed in v3.0.2
